Makefile

```diff
@@ -9,5 +9,5 @@ gdeploy:
 generate:
 	go generate ./...
 	cd web && pnpm clienttypegen 
-	pnpm prettier  --write **/openapi.yml
+	pnpm prettier  --write **/**/openapi.yml
```

Comment:
@jordiup is this intentional?
did you mean to add this to target the accesshandler/openapi.yml file?

maybe you need both of these

Reply:
Hmmmm, I was noticing a git diff on one of the openapi files (which was untouched), so I thought this would work as a better catch-all for prettier
<img width="472" alt="image" src="https://user-images.githubusercontent.com/21688404/196871661-02b032a2-fbbc-4660-a70c-55e94b9d8628.png">
Seems like the current syntax works fine, perhaps the styling just wasn't committed 🤷 
Will revert